### PR TITLE
Switch to debian:jessie image away from google/debian

### PR DESCRIPTION
**Pull request by sttts**
The latter was broken.

Fixes https://github.com/kubernetes/publishing-bot/issues/75

### CODE REVIEW

**Comment by sttts** [comment]
Smoketest is running... will report back.

File: Dockerfile [hunk 1]
```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM google/debian:jessie
+FROM debian:jessie
 MAINTAINER Chao Xu <xuchao@google.com>
 RUN apt-get update \
- && apt-get install -y -qq git=1:2.1.4-2.1+deb8u5 \
+ && apt-get install -y -qq git=1:2.1.4-2.1+deb8u6 \
```
Comment (zhouhaibing089) [inline Dockerfile]: I did the same. 👍

**Comment by zhouhaibing089** [comment]
Tested locally. LGTM

**Pull request merged**